keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -454,7 +379,6 @@ type JWTAuthenticator struct {
         // Claim must be a string or string array claim.
         // Expression must produce a string or string array value.
         // "", [], missing, and null values are treated as having no groups.
-        // TODO: investigate if you could make a single expression to construct groups from multiple claims. If not, maybe []PrefixedClaimOrExpression?
```

Comment:
We have a unit test for this: https://github.com/kubernetes/kubernetes/blob/b693f09d544b17c82af4437bd93ea0b165e2622e/staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc_test.go#L2268